frontend/packages/console-shared/src/components/quick-search/QuickSearchList.tsx

```diff
@@ -123,7 +123,12 @@ const QuickSearchList: React.FC<QuickSearchListProps> = ({
                       wrapModifier="truncate"
                       key={`${item.uid}-name`}
                     >
-                      <span className="ocs-quick-search-list__item-name">{item.name}</span>
+                      <span
+                        className="ocs-quick-search-list__item-name"
+                        data-test={`item-name-${item.name}-${itemType}`}
```

Comment:
Not used `data-test` attribute? Is it needed for another PR or could we drop it?

Reply:
Not needed.